responders/src/UI/embc-responder/src/app/feature-components/wizard/profile-components/security-questions/security-questions.component.scss

```diff
@@ -42,8 +42,16 @@ p {
   text-align: left;
 }
 
+::ng-deep .mat-select-panel {
+  max-width: 100% !important;
+}
+
+::ng-deep .mat-form-field-infix {
+  width: fit-content !important;
+}
+
 @media (max-width: 920px) {
   .col-md-6 {
     width: 100%;
   }
-}
```

Comment:
Git requires a new line at end of file (in order to be able to move live numbers around)   This is showing a No New Line issue.

Reply:
resolved